terraform/telemetry.py

```diff
@@ -0,0 +1,39 @@
+from google.cloud import pubsub_v1
+from datetime import datetime
+import sys
+
+project_id = "stackdriver-sandbox-230822"
+topic_id = "telemetry"
+
+publisher = pubsub_v1.PublisherClient()
+# The `topic_path` method creates a fully qualified identifier
+# in the form `projects/{project_id}/topics/{topic_id}`
+topic_path = publisher.topic_path(project_id, topic_id)
+
+# define the following helper functions
+def get_datetime_str():
+    # send current date and time for end of installation script
+    now = datetime.now()
```

Comment:
All the dates should be in UTC format to mitigate time differences associated with multiple users located across the world and try to create sandboxes